scripts/deploy.ts

```diff
@@ -11,11 +11,12 @@ async function main() {
     forwarderFactory: ''
   };
 
-  const feeData = await ethers.provider.getFeeData();
+  // const feeData = await ethers.provider.getFeeData();
 
   const eip1559GasParams = {
-    maxFeePerGas: feeData.maxFeePerGas,
-    maxPriorityFeePerGas: feeData.maxPriorityFeePerGas
+    maxFeePerGas: 10000000000,
+    maxPriorityFeePerGas: 10000000000,
+    gasLimit: 3000000
```

Comment:
this script doesn't contain nonce handling ?

Reply:
no, should I add it?
I'll resolve the conflicts since master contains nonce handling.